src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,205 @@
+package duke;
```

Comment:
I see that you have actually renamed your ChatBot to Linus! It's a nice name! Might it be better to change your main class to Linus to avoid confusion with Duke?